src/ducks/resource.js

```diff
@@ -0,0 +1,107 @@
+import apiClient from 'panoptes-client/lib/api-client';
+
+// Action Types
+export const FETCH_RESOURCE = 'FETCH_RESOURCE';
+export const FETCH_RESOURCE_SUCCESS = 'FETCH_RESOURCE_SUCCESS';
+export const FETCH_RESOURCE_ERROR = 'FETCH_RESOURCE_ERROR';
+export const CREATE_TRANSLATION_SUCCESS = 'CREATE_TRANSLATION_SUCCESS';
+
+// Reducer
+const initialState = {
+  original: [],
+  translation: null,
+  error: false,
+  loading: false,
+};
+
+const resourceReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case FETCH_RESOURCE:
+      return Object.assign({}, initialState, { loading: true });
+    case FETCH_RESOURCE_SUCCESS:
+      return Object.assign({}, state, { original: action.payload, loading: false });
```

Comment:
What is `original` supposed to store? Given the name, I was expecting to see in here only the original in english. However I see allI the clones with different `language` flags I created in some early tests. I'm a bit confused.